app/models/changeset/pull_request.rb

```diff
@@ -3,11 +3,14 @@ class Changeset::PullRequest
   RISKS_SECTION = /#+\s+Risks.*\n/i
 
   # Matches URLs to JIRA issues.
-  JIRA_ISSUE = %r[https://\w+\.atlassian\.net/browse/[\w-]+]
+  JIRA_ISSUE_URL = %r[https?:\/\/[\da-z\.\-]+\.[a-z\.]{2,6}\/browse\/[A-Z][A-Z\d]+-\d+]
```

Comment:
The original idea is the if an absolute url is entered, we should always respect it. It would be surprising to the users if they entered a full url and the final url presented on the UI is different from what they entered.

I agree that one org using multiple JIRA instances is not common, but it does happen in large organizations, where they do it because JIRA was not able to scale that well. Given that we are limiting the support to one BASE_URL in the configuration at the moment, I just hope that we could at least have some workaround (i.e., asking users to always enter full urls) to this problem.